orttraining/orttraining/python/training/onnxblock/README.md

```diff
@@ -0,0 +1,82 @@
+## Getting Started
+
+This is a simple guide on how to generate the training graphs using the offline tools.
+
+### The Training and Eval Graph
```

Comment:
Add a section above this which lists all the artifacts that need to be generated in the offline step including whether they are required or optional.